### PR TITLE
eunchong_test_FullRequest

### PR DESCRIPTION
**Pull request by dmsvk01**
test입니다

### CODE REVIEW

File: myapp/lib/main.dart [hunk 1]
```diff
@@ -1,116 +1,23 @@
...
-  const MyHomePage({Key? key, required this.title}) : super(key: key);
-
-  // This widget is the home page of your application. It is stateful, meaning
-  // that it has a State object (defined below) that contains fields that affect
-  // how it looks.
-
-  // This class is the configuration for the state. It holds the values (in this
-  // case the title) provided by the parent (in this case the App widget) and
-  // used by the build method of the State. Fields in a Widget subclass are
-  // always marked "final".
```
Comment (zhoho) [inline myapp/lib/main.dart:23]: What is it?

**Review by zhoho — CHANGES_REQUESTED** [review]
test

**Comment by zhoho** [comment]
good job bro

File: myapp/lib/main.dart (hunk 1 above)
Comment (dmsvk01) [reply myapp/lib/main.dart:23]: .

**Pull request closed without merging**